src/compiler/types.ts

```diff
@@ -345,17 +349,31 @@ module ts {
         // error.
         ThisNodeHasError = 1 << 5,

```

Comment:
Is there a reason we use left-shift here instead of a hexadecimal value for the flags like we do everywhere else?